.changelog/685.txt

```diff
@@ -0,0 +1,6 @@
+```release-note:feature
+ Add optional azure config block for hvn_route resource to support hub and spoke architectures, `azure_config`.
+
+ Add optional `next_hop_type` parameter.
```

Comment:
Nit: add a description for what this does since it will be in the notes or link to an explanation.